conference/react-js/src/constants/route-url.js

```diff
@@ -1,14 +1,19 @@
-export const baseUrl = '/react';
-export const root = `${baseUrl}/`;
-export const past = `${baseUrl}/past`;
-export const history = `${baseUrl}/history`;
-export const upcoming = baseUrl;
-export const forgotPassword = `${baseUrl}/forgot-password`;
-export const talks = `${baseUrl}/talks`;
-export const signUp = `${baseUrl}/sign-up`;
-export const account = `${baseUrl}/account`;
-export const manageUser = `${baseUrl}/manage-user`;
-export const myTalks = `${baseUrl}/my-talks`;
-export const myEvents = `${baseUrl}/my-events`;
-export const myInfo = `${baseUrl}/my-info`;
-export const conference = `${baseUrl}/conference`;
+import path from 'path';
+
+const { resolve } = path;
+
+export const root = '/react/';
+
+export const upcoming = root;
+export const history = resolve(root, 'history');
```

Comment:
I am sure `resolve` function is aimed to resolve filesystem paths, not URLs.
replace it with template string, or write your `resolve` function for such functionality.